src/index.jsx

```diff
@@ -89,56 +94,32 @@ class BerbixVerify extends React.Component {
   }
 
   baseUrl() {
-    const { baseUrl, environment } = this.props;
+    const { baseUrl } = this.props;
     if (baseUrl != null) {
       return baseUrl;
     }
-    switch (environment) {
-      case "sandbox":
-        return "https://verify.sandbox.berbix.com";
-      case "staging":
-        return "https://verify.staging.berbix.com";
-      default:
-        return "https://verify.berbix.com";
-    }
+    return "https://verify.berbix.com";
   }
 
   frameUrl() {
     const {
       overrideUrl,
       version,
-      clientId,
-      role,
-      templateKey,
-      email,
-      phone,
-      continuation,
       clientToken,
       showInModal,
+      showCloseModalButton
     } = this.props;
     if (overrideUrl != null) {
       return overrideUrl;
     }
-    const token = clientToken || continuation;
-    const template = templateKey || role;
+    const token = clientToken;
     var options = ["sdk=BerbixReact-" + SDK_VERSION];
```

Comment:
Let's switch these over to use template strings, for consistency.

Reply:
agreed